python_modules/dagster/dagster/_core/storage/dagster_run.py

```diff
@@ -609,6 +609,22 @@ def __new__(
             end_time=check.opt_float_param(end_time, "end_time"),
         )
 
+    @property
+    def create_time(self) -> datetime:
+        return self._asdict()["create_timestamp"]
+
+    @property
+    def update_time(self) -> datetime:
+        return self._asdict()["update_timestamp"]
+
+    @property
+    def start_timestamp(self) -> Optional[float]:
+        return self._asdict()["start_time"]
+
+    @property
+    def end_timestamp(self) -> Optional[float]:
+        return self._asdict()["end_time"]
```

Comment:
there should probably be a comment somewhere in here about this time / timestamp flip flopping

should we like do serdes stuff to "correct" the model to the right way and then add the back compat accessors as deprecated?